HaulageApplication/HaulageApp/ViewModels/ManageCustomersViewModel.cs

```diff
@@ -19,6 +18,15 @@ public ManageCustomersViewModel(HaulageDbContext dbContext)
         LoadAllCustomers();
     }
     
+    public void ApplyQueryAttributes(IDictionary<string, object> query)
+    {
+        if (query.ContainsKey("reload"))
```

Comment:
Sure - I think if more places were editing the customer, then I would use a different method, such a loading on appear / a different lifecycle method, because I agree / understand what you're saying about this being more fragile. Right now this would just introduce more querying of data and as customers aren't expected to change rapidly, I don't think this is necessary (i.e. to have that many refreshes).